lab_1/main.py

```diff
@@ -51,7 +89,19 @@ def create_language_profile(language: str, text: str, stop_words: list) -> dict
     :param stop_words: a list of stop words
     :return: a dictionary with three keys – name, freq, n_words
     """
-    pass
+    if (not isinstance(language, str)
+            or not isinstance(text, str)
+            or not isinstance(stop_words, list)):
+        return None
+    # use function remove_stop_words
+    tokens = remove_stop_words(tokenize(text), stop_words)
+    # use function calculate_frequencies
+    frequency_dictionary = calculate_frequencies(tokens)
+    # find the number of tokens in the dictionary
+    n_words = len(frequency_dictionary.keys())
+    # create language profile
+    profile = {"name": language, "freq": frequency_dictionary, "n_words": n_words}
+    return profile
```

Comment:
you can return just here `{"name": language, "freq": frequency_dictionary, "n_words": n_words}`

Reply:
Done